Addressing Genart review (#37)
ietf-rats-wg/draft-ietf-rats-network-device-subscription: draft-ietf-rats-network-device-subscription.md

# Privacy Considerations

The Privacy Considerations of {{-rats-riv}} (Remote Integrity Verification of Network Devices Containing Trusted Platform Modules) apply.

> Additionally, the Security Considerations from {{RFC8641}} (Subscription to YANG Notifications for Datastore Updates) outline how internal structures or capabilities about the system can leak, which can have an impact in personally identifiable information (PII).

Reviewer: ```suggestion
Additionally, the Security Considerations from {{RFC8641}} (Subscription to YANG Notifications for Datastore Updates) how information about the system's internal structures or capabilities can be leaked, which could impact personally identifiable information (PII).
```

Reviewer: Also, why are security and privacy considerations capitalised?

There are no additional privacy considerations introduced by this document.

# Security Considerations

The Security Considerations of {{-charra}} and {{-rats-riv}} apply.

> Additionally, the Security Requirements {{Section 4.2.5 of RFC7923}} and the Security Considerations {{Section 5 of RFC7923}} from RFC7923 (Requirements for Subscription to YANG Datastores) apply.

Reviewer: Missing parentheses.  (Also, ditto, unclear extra capitalisation.)

```suggestion
Additionally, the Security Requirements ({{Section 4.2.5 of RFC7923}}) and the Security Considerations ({{Section 5 of RFC7923}}) from RFC7923 (Requirements for Subscription to YANG Datastores) apply.
```
